### ISSUE TITLE
Better support for Tensorflow and Keras

### ISSUE THREAD

**Issue by chengchihhong** [dependable-ai/nn-dependability-kit#15]
As title. Currently the framework tries to be platform independent by using numpy for potential support of Tensorflow. Further experiments and refactoring shall be executed conducted to fully support Tensorflow.

**Comment by chengchihhong** [comment]
Note that the weight matrix in tensorflow (I only tested from models loaded from .pb file) and pytorch are different.

pytorch [dimension of output, dimension of input]
tensorflow [dimenson of input, dimension of output]

So currently I change the axis of tensorflow before feeding into the intermediate net format for analysis.

**Comment by chengchihhong** [comment]
Preliminary support for extracting MLP from tensorflow model done.
